### PR TITLE
Updating poc styling

### PR DESCRIPTION
**Pull request by jpa399**
_Pull requests into Flux require the following. Submitter should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:. Reviewers should complete every item on the bulleted list when reviewing._

Updated several poc placeholder styles to match 15.5 mockups, including: removing the "done" box (but keeping the functionality there for future changes that bring it back), moving the expand arrow to the left side and changing its orientation (from up and down to left and down), standardizing the weights, sizes, and colors of words and underlines, removing the "#" from the field titles, restyling the radio buttons and checkboxes (multiselect buttons are more separated than single select), preventing overlap of patient info in the patient control panel when switching to ella, adding an arrow in the dashboard to indicate which tab in poc is being viewed, and centering the field titles with the top row of buttons.  Also deleted extra code that pertained to the summary section deleted in my previous pull request.  Fixed the bug where the patient summary info overlaps after switching patients.  Finally, changed the single select button onclick function so that they deselect if they were already selected.  This pull request combines several Jira tasks:
https://jira.mitre.org/browse/ASCODCP-1993
https://jira.mitre.org/browse/ASCODCP-1405
https://jira.mitre.org/browse/ASCODCP-2183
https://jira.mitre.org/browse/ASCODCP-2014

## Submitter:

**Running the Application**

- [x] Manually tested in Chrome (primary browser for testing)
- [ ] Manually tested in IE/Safari (verify app loads and basic feature testing)
- [x] Manually tested in Chrome in iPad mode (at minimum, viewing patient record)

**Task**

- [x] Branch implements task as expected
- [ ] Key application features are updated as needed for task and tested (see: [Key Application Features](https://github.com/FluxNotes/flux/wiki/Key-Application-Features))
- [x] Task specific definition of done (as described in the Jira task) is met, if applicable.

**Documentation**

- [x] This pull request describes why these changes were made
- [x] Recognizes any potential shortcomings/bugs in the description 
- [ ] Pre release script is updated on the [Pre Release Testing Script wiki page](https://github.com/FluxNotes/flux/wiki/Pre-Release-Testing-Script) (add/remove single features to test, do not run through full script)
- [ ] Update the list of key features with new features on the [Key Application Features wiki page](https://github.com/FluxNotes/flux/wiki/Key-Application-Features)
- [ ] Document any new APIs/extension points
- [ ] Additional technical components and libraries are documented and added to [wiki](https://github.com/FluxNotes/flux/wiki/About-Flux-Notes) as needed

**Code Quality**

- [x] Code style guide is followed (see: [Code Style Guide](https://github.com/FluxNotes/flux/wiki/JavaScript-and-HTML-Code-Style-Guide))
- [x] Code is commented

**Tests**

- [x] Existing tests passed
- [ ] Added backend tests for new functionality added

## Reviewers:

- Check that code is maintainable and reusable. Code reuses existing code and infrastructure where appropriate.
- Check the PR and code accomplishes the task's purpose.
- Check that new tests appropriately test the new code, including edge cases. Other existing tests pass.
- Verify Pull Request checklist is correctly completed by submitter.
- Try to break the code

### CODE REVIEW

**Comment by jpa399** [comment]
[JPA1] rebasing off of master somehow removed my changes to the expand icons. Working on that.

**Comment by jpa399** [comment]
[JPA1] resolved

File: src/notes/FillPlaceholder.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Grid from 'material-ui/Grid';
-import Checkbox from 'material-ui/Checkbox';
+// import Checkbox from 'material-ui/Checkbox';
```
Comment (sgober) [inline src/notes/FillPlaceholder.jsx]: do we still need this in here?

File: src/notes/FillPlaceholder.jsx [hunk 2]
```diff
@@ -380,16 +353,16 @@ export default class FillPlaceholder extends Component {
     }
 
     renderCheckbox = () => {
-        const { done } = this.state;
+        // const { done } = this.state;
```
Comment (sgober) [inline src/notes/FillPlaceholder.jsx]: can we remove these comments as well?

Comment (sgober) [reply src/notes/FillPlaceholder.jsx]: would help if I read the PR description...

**Comment by jafeltra** [comment]
I just started reviewing this and it looks really good! A few things to potentially look into:
[JA1] I don't think there should be any border radius on the "select one" buttons. Then if possible, the outer edges have a border radius, but that was a low priority item, so we don't get to that, it's okay.
[JA2] The mock ups don't show any drop shadows on the buttons. Do you think we can get rid of those?
[JA3] Is it possible to keep the expand arrow on in line with the shortcut name on the lefthand side? When you have multiple toxicities, it becomes centered vertically, but expanding the section moves the arrow to the top. I don't think this was in Dan's list of styling fixes, so if it isn't trivial to fix, we can talk about if we want to actually change anything.

**Comment by dreeves4321** [comment]
[DR1] button text is styled as black with a transparency. should be opaque with bodyblack.
[DR2] when toxicities expand, the summary information indents and drops a line
regarding [JA3], I agree that the arrow should remain in the top row, always. It's very distracting to have it jump around, and its function becomes very confusing.
[DR3] Not sure what's in my mockups, but the horizontal line separating toxicities would make more sense left justified with the summary elements ("Adverse Events") rather than the title "Toxicitiy".

**Comment by jpa399** [comment]
[DR1] and kind of [JA1] resolved!

**Comment by jpa399** [comment]
[DR2] resolved!

**Comment by jpa399** [comment]
[JA3] resolved!

**Comment by jpa399** [comment]
[DR3] resolved!

**Comment by jpa399** [comment]
[JA2] resolved!  All current comments should now be resolved.

**Comment by jafeltra** [comment]
JA1, 2, and 3 all look good! One other thing I noticed:
[JA4]: Not all of the single choice buttons current deselect their values. For example, on the Adverse Event and Grade buttons on the toxicity placeholder, I can't deselect the values.

**Comment by jafeltra** [comment]
[JA5] Instead of using `placeholder.shortcutName.slice(1)` to get rid of the # on placeholders, you should write a get displayText function in the placeholder class (Placeholder.jsx) that you can then call from all those places. The display text function can handle getting rid of #.

**Comment by jpa399** [comment]
[JA5] resolved!  Workin on [JA4]

**Comment by jpa399** [comment]
[JA4] resolved!

**Comment by jafeltra** [comment]
JA4 and JA5 look good now! Two more small things:
[JA6]: Deselecting the adverse event in toxicity doesn't clear the search bar. Do you think it should?
[JA7]: The summary line for a second toxicity seems to have disappeared.

**Comment by jpa399** [comment]
[JA6] resolved!

**Comment by jpa399** [comment]
[JA7] resolved!

**Comment by jafeltra** [comment]
The summary placement looks good! But I did notice that the summary values for a second toxicity are the same as the first. They should function independently from each other.

**Comment by jpa399** [comment]
all fixed!

**Comment by jafeltra** [comment]
This looks great! All the JA's are addressed! It just needs a final rebase!

**Comment by jpa399** [comment]
rebased!
